### ISSUE TITLE
Size and nature of the dataset.

### ISSUE THREAD

**Issue by Sreerag-ibtl** [qjadud1994/CRNN-Keras#21]
I have a 25451 images in the train and 2000 images in the test set. The problem I am facing is the train and validation loss is high in every epochs. In my dataset a few images are of plates with two lines. So I would like to get a suggestion about the dataset size and it's nature. TIA

**Comment by mzeynali** [comment]
Hi @Sreerag-ibtl,
In my opinion, the number of dataset is small. because i tested this problem, i've solved this problem with  number of large dataset. around 500k images.

**Comment by Sreerag-ibtl** [comment]
Thanks for the reply. I have a fair amount of synthetic images. I tried a different implementation of the CRNN a while ago, with those. But the prediction was not better.
